de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/EditorManager.java

```diff
@@ -769,6 +773,16 @@ void generateTextEdit(int offset, String newText, String replacedText,
             return;
         }
 
+        if (fileReplacementInProgressObservable.isReplacementInProgress()) {
+            if (LOG.isTraceEnabled()) {
```

Comment:
Why do you use `isTraceEnabled` ? This is only useful if you want to avoid to execute an expensive task even if Trace is disabled. In your cases you simply concat strings.

Reply:
Maybe it might not be necessary in this case as it should not happen that often, but I included this check in reaction to a comment by @srossbach on #262:

> Just for the future, generating the log message even it is not logged requires 3 string concats and a toString method call. I do not think it will haven that often and consumes much CPU time but you should really think about
> 
> if (LOG.isTraceEnabled()) … before such log statements

Reply:
Ok, I had the interface of slf4j and log4j2 in mind that supports a method `trace(String formatString, params ...)` that evaluates the format string only if the corresponding log level is set, but this is currently not supported in our environment.